gap/digraph.gi

```diff
@@ -381,6 +381,73 @@ function(graph)
   return StructuralCopy(graph!.vertexlabels);
 end);
 
+# edge labels
+InstallMethod(DigraphEdgeLabel, "for a digraph, a pos int, and a pos int",
+[IsDigraph, IsPosInt, IsPosInt],
+function(graph, i, j)
+    local p;
+    DIGRAPHS_InitEdgeLabels(graph);
+    p := Position(graph!.adj[i], j);
+    if p <> fail then
+        return ShallowCopy(graph!.edgelabels[i][p]);
+    else
+        ErrorNoReturn("DigraphEdgeLabel: vertex"
```

Comment:
The whitespace in this error message is still incorrect - it looks like "DigraphEdgeLabel: vertex[ 3, 4 ]..." (no space between 'vertex' and the last).